Core/Core/Store/Store.swift

```diff
@@ -143,25 +142,24 @@ public class Store<U: UseCase>: NSObject, NSFetchedResultsControllerDelegate {
     }
 
     public func getNextPage(_ callback: ((U.Response?) -> Void)? = nil) {
-        getNextQueue.sync {
-            guard let next = next else {
-                performUIUpdate {
-                    callback?(nil)
-                }
-                return
+        guard let next = next else {
+            performUIUpdate {
+                callback?(nil)
+            }
+            return
+        }
+        self.next = nil
+        let useCase = GetNextUseCase(parent: self.useCase, request: next)
+        print("get next")
```

Comment:
If this works, will need to remove this.

Reply:
dang. yeah, I'll let see if it builds then I'll remove this then we'll have 2 builds to verify these changes.

Reply:
👍